### PR TITLE
#65 Adding README and declarative resources for Hello World example

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: examples/apache-maven/custom-pipelines-hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Place a purchase order like so:
+
+```bash
+export DECODABLE_API_KEY_BASE64=$(echo -n 'my very secret API key' | base64)
+
+curl -X POST localhost:8080/v1alpha2/connections/9b043743 \
+  -H "Content-Type: application/json" \
+  -H "Accept: application/json" \
+  -H "Authorization: Bearer ${DECODABLE_API_KEY_BASE64}" \
+  -d @purchase-order.json
```
Comment (gunnarmorling) [inline examples/apache-maven/custom-pipelines-hello-world/README.md:35]: This follows the instructions from the docs, but doesn't actually work (see internal discussion thread). Waiting for clarification here.

File: examples/apache-maven/custom-pipelines-hello-world/decodable-resources.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+metadata:
+    name: customer-processor
+spec_version: v2
+spec:
+    type: JAVA
+    job_file_path: target/custom-pipelines-hello-world-0.1.jar
+    properties:
+        flink_version: 1.16-java11
+    execution:
+      active: false
```
Comment (hpgrahsl) [inline examples/apache-maven/custom-pipelines-hello-world/decodable-resources.yaml:67]: is it on purpose that the pipline isn't activated by default in this example? connection gets activated right away and so could the pipeline or am I missing something maybe?

Comment (gunnarmorling) [reply examples/apache-maven/custom-pipelines-hello-world/decodable-resources.yaml:67]: Ah yeah, good catch, will change that.

Comment (gunnarmorling) [inline examples/apache-maven/custom-pipelines-hello-world/decodable-resources.yaml:67]:
```suggestion
      active: true
```

**Review by hpgrahsl — APPROVED** [review]

**Pull request merged**